overrides/home.html

```diff
@@ -36,7 +36,7 @@ <h3 class="trusted-by">Trusted by</h3>
     <div class="md-grid md-typeset">
       <h1>Need to know more?</h1>
       <p class="normal-text">
-        Learn about how Knative works with Kubernetes and the underlying ideas behind containers<br>
+        Learn about how Knative works with Kubernetes and the underlying ideas behind containers <br class="br">
```

Comment:
```suggestion
        Learn about how Knative works with Kubernetes and the underlying ideas behind containers
```

Instead of setting `display: none` for this class, can we just remove the tag?